runtime/queries/rust/highlights.scm

```diff
@@ -287,11 +290,11 @@
   name: (identifier) @function.macro)
 (macro_invocation
   macro: [
-    ((identifier) @function.macro)
+    ((identifier) @keyword.directive)
     (scoped_identifier
-      name: (identifier) @function.macro)
+      name: (identifier) @keyword.directive)
   ]
-  "!" @function.macro)
+  "!" @keyword.directive)
```

Comment:
Invocations shouldn't be `keyword.directive`, since they're not keywords (they're macro invocations = `@function.macro`)

Reply:
Yeah, i've thought about `keyword.directive` in a more broad sense, like the place of interest for the preprocessor, while it's purpose is to match preprocessor-specific keywords i.e. `#if #define`, makes sense.

Reply:
i'll rebase request to include only commit about `macro_rules!` -> `@keyword.directive`